tiatoolbox/tools/patchextraction.py

```diff
@@ -53,29 +52,86 @@ class PatchExtractor(ABC):
         input_img(ndarray, WSIReader): input image for patch extraction.
           input_image type is ndarray for an image tile whereas :obj:`WSIReader`
           for an WSI.
-        patch_size(Tuple of int): patch size tuple (width, height).
-        resolution(Tuple of int): resolution at which to read the image.
+        patch_size(tuple(int)): patch size tuple (width, height).
+        resolution(tuple(int)): resolution at which to read the image.
```

Comment:
Need to edit the doc as the code support both int and tuple(int)

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/97/commits/f85a61e9650b7dad8ffa73f6eafd37c5d18e5807